src/core/agent/services/multiSigService.ts

```diff
@@ -310,17 +311,18 @@ class MultiSigService extends AgentService {
   @OnlineOnly
   async joinMultisigRotation(notification: KeriaNotification): Promise<string> {
     const msgSaid = notification.a.d as string;
-    const notifications: MultiSigExnMessage[] = await this.props.signifyClient
-      .groups()
-      .getRequest(msgSaid)
-      .catch((error) => {
-        const status = error.message.split(" - ")[1];
-        if (/404/gi.test(status)) {
-          return [];
-        } else {
-          throw error;
-        }
-      });
+    const notifications: InceptMultiSigExnMessage[] =
```

Comment:
How is a multi-sig rotation event `InceptMultiSigExnMessage`? It should be a rotation not an inception

Reply:
@Sotatek-BaoHoanga Some of these are meant to be `InceptMultiSigExnMessage` and some are meant to be `RotationMultiSigExnMessage`. Please have 2 notification types, and set incept for any inception messages when creating the multi-sig and rotation for when rotating...